### ISSUE TITLE
avoid setting background as tiles

### ISSUE THREAD

**Issue by kakaroto** [kakaroto/R20Converter#4]
If we find a graphics that doesn't match 100% but matches, let's say 95% of the size and is the only one that meets the criteria, then set it as background with a position shift.
Might be a bit tricky if the size doesn't match though, would need work through that somehow.

**Comment by tposney** [comment]
No, please leave them as tiles - it is much more flexible.

**Comment by kakaroto** [comment]
Yeah, it's more flexible, but I know Atropos said that there is a reason/advantage to having them as the actual background.. I can't remember what it was, but regardless, one issue I have is that selecting a tile brings it to the front, so when your background is a tile, it makes things really difficult to work with sometimes.
Regardless of that, I understand your preference and I'll just make a `--background-as-tile` option for those who prefer them all to be tiles.
